public/logo_mellow.svg

```diff
@@ -0,0 +1,3 @@
OV1VrBBhU+L/XttBZVUNnHvzOJ7IpefRAy0Dp6KIAi1y8XeZAY3fFfYMt93BLedYPSxqePwcRnHAg3MvzCD9roI/xXVY5O/jqTAcxAmiCw/V8ycxf0LLboth8E87faqC1B0K2qyQazp5TvGr1sg7XHqNhuAEMP3wf61J3VXf2dbrwnDsN9/eyDBQfK59sCJLbTIe35+nSPVNgKWYCP1fmB/6m5aSNlgx+Rvkh/F21NOftWHeyv8NWHX8GgqYIcYVhK4m4HlzHl9zbGvjyNVTtaKCFtchPk+h0crnpUbhY5ykkYjED0yPhEL3iaIC5UENngnAcZNsfqvxMeXvfRXjA3+Ee05mmyV2xtyC7qA/bKoUXrD4Z/m3yiPE9J+4O980WvsY/aWjbLPUqDIJ7TUaw/RB0OZoLVcdtk56Py5mCs4WuYvl8BxuKiEXBDVEnY3eEbNtJNEy2jBXAQPa7EcjVuqnbm3wj/OZSwXkFDZ25MEErYJgyFbA6x/JWd90n8crWjxiOaubA4JHRApJmy29aly/gQXD9cPh3a8PLwte7hgneYnyfd5ZBVz09aXN/4hcNe/yPETyLOFugdYDlWr6XXBL+bfriH+7Jx6sGWqcgZRsn0DB9kJoJLrW//cc1BtX01udvbJcoA2ZwfhbfYKaYvqUsE/VoRz74MdW54Bml+0oMle7rAl9a0dnhObA8uoCfBz8sXPtPzbMx56ncEYAbLBY26Mr1Rc3BE37fAi9E3VcHs4/jaC0M0mxbZQI3QHInmc4GWd/05OMyA60PHe5XQYzSWWBZ/vItZXbPn0K5dDAtf3lzPVw+w4GW7azpBcO9rgf6q0tKP1lDrwGs4+1hAMrL3Zs8w7e48/wU4W+dDPKoIpxMeBjuO1C4RudjPNGTNneC+QH+fYSG7jeedGWIxhxT8DkfB+JyWxURxZuPZ5DpyKdiOS834Q2Bxm6a61oIdZFdbD9/wOsWxhsdLYmLO0xUmyHBVHhptqM1u6fw1eL7uTs0NGeBY7X4txdgMvhGuvCX3jwpplVXVds3i+AwuE88hzV9VZf3oN1VWHnwZfNDGrrY/E8MXnycjq6C+vLCOfQg4e/dDbL6xPouY6BTEXwJfCE6eOi2i2sKDjl7+T7AtPzlbS1HWm9Y6FzhQMPkjbKNw/0dBOODuLK43UB3FtTj3Ivg3N5OiHIR99NtNPl5f/Fc/vI96VXS3/+mkclnwLUX0p6KQeJjNLSxaUnEs0zVUUZryTC5k/S77sze50vf0uIUMx0pWzth23S35vehwnO8M1O6LH9dHb5tk94W2cPAyKML67re4f42wmCQK3Tr3Mi2cpTPxjf30pIPvnVHYT6GEO5Ndr/it28VtH2OVNoJxip5ua+KbnrVg7aMR4DGfHC3bAVOKFuCqyF/xhiFs00LgwXY5SXOwU8YdPV9dkXKxpfsc+BIrRW4aG4hLfvvxn4QbMtfVXYGFXpY6HzumKBqE8Kbs7ngfKBaCQxT0G1UXOeLZsKSTbUXV602fI6d+ER9XPHb5sQBt7CgM9XRnsq/GLsEvsDhwzDG4BO9j+YeH68s/rI4WsoyQn0R6eJ6xfZvjxXqn6CLtePNJ7bQZP6ueGz8D9XcbzPwNBH8i9fRXHOIgq6vAUZGa6HTN1f8rttjObmdCcdGulWJysLsGtjcUYhT1uUKHqWgbwv/U0HU/RA4dxwCIWRZuOUODrYInc1hSct9HHxlxV9KKlqYPF7HC/efACupi+Fs/1Q4cyVljbQtf33O0AZbaMn7DhNUib3usdzTRDgq0sXAMvDekmlj65ky2BvzJbduAKuKSqmWsiqcL3xtdFAl43LxUush7FFNx1FDFPR996ui0eVkh+t1ULmKujrX8/BM7r+9sYKWyZNOB54OaabhI0GGafn7q8chejfLRNVZ+3SQv34LLJkC2wjuey7xp/9TLE0wWF44JzT5R6vSerh4xPAlu21185JE+14H2n0EQ4gt55F8vPGdZ1Lzg4R7bSk7bZD7ruLoF34DXCtmZ1AFYvi+hHgZzvc9jki9cbyCUbH5GFzGGej45BfmkNO+mN6cvQRnDdc4Q/nLjYlw4QN4nsPyXs4s6DKZuDHCtqJgiuBm21dVjCt2Od/tKNH/1OEeDtHz61CP+3SQt1ousa/86OlM6e9yUgXfFdZojz5Co6fEpNx8limHG+h8jaztuYhER2feFpcxvhndRZdDXzM9N72/5eh/2kWSx+Sm2ByMJRVYtrlAdIS3ee/sIVxri5YSIS61Gy0uogysyzxS5xdE9IsOt0krrjUM1/YQtkaqLITNpHHg6uW1jvCBcPWiQ8N0DuqzZpcHodxWIPAoZv9XZTQ4AX77BBnytaHEqymOlmNL4YhkuMP1Il4WnqWb4GsJ+1+f/MdrC7RN2R352eVURL7a9SV99TNcu6eQdXKch/HHBdsIPMwyXD9AuM50pCN6fdlOPRYRUvBMcrQLkGBa/tr8M2XIeyWxDfIj9SesI9AS6+EsJwx8l+WkCfJyWGVkWQiMY+PhmhmIEDYmySO/5btKg8bV91gEj6pSLVG3EfZ6j3qkB5UhnlF/JA3WJmDxFJep8rIzFMQXn/zF3kwINpnosI/sJKQJmq5JOrC2YIXObDmH5STXHyCH7PClxHRYPrl+oZpojgvmgFOA7F2DhejFco+wPAy1/2HgbyoejE+GAO57pOz6PsskHUTHiLth+Sq+8MYgImM4Wggrpgbon1HSMs8WHO8COfXoW8KLjLfbiFkjbGgvZVa8B+KUxdBAnxjnTaVaTI/DPne84HXWgMiOiEZTGOAXgrV1ZzDI9EKc/7SGSIWTwNLZHxzXuzjsZXxxmBSofW2AKm0yusAh/zvSoJsLnUaJLJGxqeKo5QmEI78OZ0ludNMh2giTHE2HQRAiKJaEmAzPRpeD8ET2wcjCJ8Vshlmgd2ad9ckawpG9XLKPjFjSJVulZNFqqgT04cEY+z9Y59VfYZOOBAAAAABJRU5ErkJggg=="/>
```

Comment:
This is not a proper vectorial SVG, just a PNG encapsulated inside the SVG code.

We need a proper vectorial SVG image to make sure it shows correctly with any screen resolution.